### DISCUSSION TITLE
-composite switch... Where did it go?

### DISCUSSION THREAD

**Discussion by blue-j** [ImageMagick/ImageMagick#2802] [General]
I'm trying to run some of Anthony's difference stats and my v7 IM is saying -composite is no longer valid.

`convert image1 image2 -compose Difference -composite -colorspace gray -format '%[fx:mean*100]' info:`

_compare: unrecognized option `-composite' @ error/compare.c/CompareImagesCommand/507._

[https://legacy.imagemagick.org/Usage/compare/#statistics](url)

Does anyone know how to make this work with the current version?    Thanks in advance for your attention to this matter!   : )

- J

**Comment by fmw42** [comment]
If on Windows, replace single quotes with double quotes.  Your command works fine for me on my Mac using double quotes.

**Comment by t246246** [comment]
Probably you typed `compare` instead of  `convert`.
